crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -213,11 +447,19 @@ reveal_type(True.numerator)  # revealed: Literal[1]
 reveal_type(False.real)  # revealed: Literal[0]
 ```
 
-## Bytes-literal attributes
+### Bytes-literal attributes
 
 All attribute access on literal `bytes` types is currently delegated to `buitins.bytes`:
 
 ```py
 reveal_type(b"foo".join)  # revealed: @Todo(instance attributes)
 reveal_type(b"foo".endswith)  # revealed: @Todo(instance attributes)
 ```
+
+## References
```

Comment:
is it also worth linking to https://typing.readthedocs.io/en/latest/spec/class-compat.html here?